Python Examples/TestMonitor/CreateResultsAndSteps/create_results_and_steps.py

```diff
@@ -118,6 +124,70 @@ def generate_step_data(
     return step_data
 
 
+def is_partial_success_response(response: Dict) -> bool:
+    return "error" in response.keys()
+
+
+def get_test_result() -> Dict:
+    test_result = {
+        "programName": "Power Test",
+        "status": {
+            "statusType": "RUNNING",
+            "statusName": "Running"
+        },
+        "systemId": None,
+        "hostName": None,
+        "properties":None,
+        "serialNumber": str(uuid.uuid4()),
+        "operator": "John Smith",
+        "partNumber": "NI-ABC-123-PWR1",
+        "fileIds":None,
+        "startedAt": str(datetime.datetime.utcnow()),
+        "totalTimeInSeconds": 0.0
+    }
+
+    return test_result
+
+
+def create_result() -> Dict:
+    test_result = get_test_result()
+
+    response = test_data_manager_client.create_results(results=[test_result])
+    if is_partial_success_response(response) :
+        raise Exception("The test result is not created, please check for correct result details and you have correct access for creating the new test results")
+    test_result = response["results"][0]
+    print(f"New test result is created under part number = {test_result['partNumber']} with Id = {test_result['id']}")
+    
+    return test_result
+
+
+def update_result(test_result: Dict) -> None:
+    remove_if_key_exists(dict=test_result, key="workspace")
```

Comment:
I think its worth a comment explaining why we are recommending you don't include workspace when updating a result

Reply:
Added the explanation of why we are removing the workspace field from the request body when updating the result.